doc/devel/coding_guide.rst

```diff
@@ -41,47 +41,53 @@ Documentation
 PR Review guidelines
 ====================
 
+* Be patient and `kind <https://youtu.be/tzFWz5fiVKU?t=49m30s>`__ with
+  contributors.
+
 * If you have commit rights, then you are trusted to use them.  Please
   help review and merge PRs!
 
-* For code changes (anything in ``src`` or ``lib``) two developers
-  (those with commit rights) should review all pull requests.  If you
-  are the first to review a PR and approve of the changes use the
-  github `'approve review'
+* Documentation and examples may be merged by the first reviewer.  Use
+  the threshold "is this better than it was?" as the review criteria.
+
+* For code changes (anything in ``src`` or ``lib``) at least two
+  developers (those with commit rights) should review all pull
+  requests.  If you are the first to review a PR and approve of the
+  changes use the github `'approve review'
   <https://help.github.com/articles/reviewing-changes-in-pull-requests/>`__
   tool to mark it as such.  If you are a subsequent reviewer and you
   approve, either merge (and backport if needed) or select ``'approve
-  review'``.
+  review'`` if you think further review is required.
```

Comment:
I tend to approve explicitly also as a second reviewer, even when I merge immediately afterwards. IMO this makes it more clear who the two reviewers are. Since I'm usually on the git review page anyway, it's just two extra clicks - so not much work.

I don't demand, that everybody does it, but I would appreciate it.